resources/fileTemplates/code/Magento Crontab Cronjob Registration.xml.html

```diff
@@ -0,0 +1,54 @@
+<!--
+/*
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
```

Comment:
Hi @roma-glushko 
We are not storing copyrights under templates. Could you please remove them. Thank you!

Reply:
@VitaliyBoyko it seems like a template explanation, not exactly the template. We used to add copyrights to all of such files. Please let me know if this changes the situation here 🙌

Reply:
Sorry, sure